src/config/dbConnet.js

```diff
@@ -0,0 +1,30 @@
+import { Pool } from 'pg';
```

Comment:
The name of this file is spelt incorrectly. Sorry for the nitpick, I just wanted to point it out cause you might have missed it.